### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.3 - es. 4 (7 ottobre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#275] [Esercizi M-Z]
![es.4](https://github.com/user-attachments/assets/b3077272-a0f2-4330-9b39-73a5748cd025)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/cddcfd54-d08c-4b5b-8ad1-9e2c084b99e9)
